.gitignore

```diff
@@ -1,5 +1,4 @@
-### https://raw.github.com/github/gitignore/f2ce448f2ba7a092da05482ceca99209127c0884/maven.gitignore
-
+# maven
```

Comment:
Do you want me to undo that change? it's not a huge issue from my perspective, but no ish if you'd rather have it without!